src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -284,9 +284,6 @@
   </ItemGroup>
   <ItemGroup>
     <VsixIgnoreFile Include=".vsixignore" />
-    <None Include="source.extension.vsixmanifest">
```

Comment:
Is this a duplicate include? 
Can you still open the manifest with the vsix designer?

Reply:
Yes and yes.